setup.py

```diff
@@ -60,132 +109,78 @@ def download_driver(zip_name: str) -> None:
 
 
 class PlaywrightBDistWheelCommand(BDistWheelCommand):
-    user_options = BDistWheelCommand.user_options + [
-        ("all", "a", "create wheels for all platforms")
-    ]
-    boolean_options = BDistWheelCommand.boolean_options + ["all"]
-
-    def initialize_options(self) -> None:
-        super().initialize_options()
-        self.all = False
-
     def run(self) -> None:
-        shutil.rmtree("build", ignore_errors=True)
-        shutil.rmtree("dist", ignore_errors=True)
-        shutil.rmtree("playwright.egg-info", ignore_errors=True)
         super().run()
         os.makedirs("driver", exist_ok=True)
         os.makedirs("playwright/driver", exist_ok=True)
-        base_wheel_bundles: List[Dict[str, str]] = [
-            {
-                "wheel": "macosx_10_13_x86_64.whl",
-                "machine": "x86_64",
-                "platform": "darwin",
-                "zip_name": "mac",
-            },
-            {
-                "wheel": "macosx_11_0_universal2.whl",
-                "machine": "x86_64",
-                "platform": "darwin",
-                "zip_name": "mac",
-            },
-            {
-                "wheel": "macosx_11_0_arm64.whl",
-                "machine": "arm64",
-                "platform": "darwin",
-                "zip_name": "mac-arm64",
-            },
-            {
-                "wheel": "manylinux1_x86_64.whl",
-                "machine": "x86_64",
-                "platform": "linux",
-                "zip_name": "linux",
-            },
-            {
-                "wheel": "manylinux_2_17_aarch64.manylinux2014_aarch64.whl",
-                "machine": "aarch64",
-                "platform": "linux",
-                "zip_name": "linux-arm64",
-            },
-            {
-                "wheel": "win32.whl",
-                "machine": "i386",
-                "platform": "win32",
-                "zip_name": "win32_x64",
-            },
-            {
-                "wheel": "win_amd64.whl",
-                "machine": "amd64",
-                "platform": "win32",
-                "zip_name": "win32_x64",
-            },
-        ]
-        self._download_and_extract_local_driver(base_wheel_bundles)
-
-        wheels = base_wheel_bundles
-        if not self.all:
-            # Limit to 1, since for MacOS e.g. we have multiple wheels for the same platform and architecture and Conda expects 1.
```

Comment:
is this comment still relevant? if so restore?

Reply:
Not anymore since we always output a single wheel now instead of multiple like before.